File: Modul4/Soal 2/PRAK302-2210817210010-Eric Nanda Ferdian.c
#include <stdio.h>

int main()
{
    int bil;
    scanf("%d",&bil);
    if(bil>=80)
    {
        printf("A");
    }
    else if(bil>=70&&bil<=79)
    {
        printf("B");
    }
    else if(bil>=60&&bil<=69)
    {
        printf("C");
    }
    else if(bil>=50&&bil<=59)
    {
        printf("D");
    }
    else if(bil<50)
    {
        printf("E");
    }
}

File: Modul4/Soal 2/PRAK302-2210817210010-Eric Nanda Ferdian.py
bil = int(input(''))

if bil  >= 80:
   nilai = ('A')
elif bil >= 70 :
   nilai = ('B')
elif bil >= 60 :
   nilai = ('C')
elif bil >= 50 :
   nilai = ('D')
elif bil <50:
   nilai = ('E')

print('%s' % nilai)